python/dgl/graphbolt/sampled_subgraph.py

```diff
@@ -189,6 +191,25 @@ def exclude_edges(
                 )
             return calling_class(*_slice_subgraph(self, index))
 
+    def to(self, device: torch.device) -> None:  # pylint: disable=invalid-name
```

Comment:
There are multiple duplicated implementation of to()  in graphbolt, consider create a parent class and inherit from it.

Reply:
Will do this in a new pr.